File: TransCoder_tokenized_test_set_functions/python/CALCULATING_FACTORIALS_USING_STIRLING_APPROXIMATION.py
def stirlingFactorial ( n ) :
    if ( n == 1 ) :
        return 1;
        
    e = 2.71;
    
    z = ( math . sqrt ( 2 * 3.14 * n ) * math . pow ( ( n / e ), n ) );
    
    return math . floor ( z );

File: TransCoder_tokenized_test_set_functions/java/CALCULATING_FACTORIALS_USING_STIRLING_APPROXIMATION.java
public static int stirlingFactorial ( double n ) {
  if ( n == 1 ) return 1;
  double z;
  double e = 2.71;
  z = Math . sqrt ( 2 * 3.14 * n ) * Math . pow ( ( n / e ), n );
  return ( int ) ( z );
}